pytorch/training/docker/1.13/py3/Dockerfile.ec2.gpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file should be deleted. Allowlist json files for GPU should be under cu*** folders.

Reply:
acked, will fix it in PT-1.13 release PR